data/src/test/java/org/apache/iceberg/data/TestLocalScan.java

```diff
@@ -556,8 +546,8 @@ public void testFilterWithDateAndTimestamp() throws IOException {
             required(3, "date", Types.DateType.get()),
             required(4, "time", Types.TimeType.get()));
 
-    File tableLocation = temp.newFolder("complex_filter_table");
-    Assert.assertTrue(tableLocation.delete());
+    File tableLocation = File.createTempFile("junit", null, tempDir);
```

Comment:
this should be created a temp directory rather than a temp file

Reply:
fixed